app/controllers/citizens/citizen_base_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Citizens
+  class CitizenBaseController < BaseController
+    include ApplicationFromSession
+    before_action :authenticate_applicant!
+    before_action :check_not_complete
+
```

Comment:
Don't you need to add `before_action :set_cache_buster` here?

Reply:
Thank you! I had a niggling feeling I'd missed something there ... and that was it!